### ISSUE TITLE
Import/Export pack configuration?

### ISSUE THREAD

**Issue by juju4** [zentralopensource/zentral#26]
Hello,

Any way to import an existing pack configuration file either through gui or cli?
More generally, would want to import/export queries, pack or osqueryd standalone config.

Thanks

**Comment by headmin** [comment]
Did you check out the probe feeds ? You add an osquery pack to probe feeds, then import to a new probe. Updates in the original pack will be indicated, you see a diff and decide to update your probe.

Probes can be saved as a gist, i.e. you can share this with others or just store.

**Comment by juju4** [comment]
No, I don't think so. But can I import through command line? where are the corresponding files?

File/Templates are easier to integrate in an orchestrated deployment.

**Comment by np5** [comment]
There is a django management command to add a probe feed:

`python server/manage.py add_probe_feed`

There is actually no way to export and import the full zentral configuration via files, but we are currently trying to find a solution.

**Comment by np5** [comment]
There is an [API endpoint now](https://docs.zentral.io/en/latest/apps/osquery/#apiosquerypacksslug) to import Osquery packs.
